modules/message-lane/src/lib.rs

```diff
@@ -90,19 +159,237 @@ decl_module! {
 		/// Deposit one of this module's events by using the default implementation.
 		fn deposit_event() = default;
 
+		/// Block initialization.
+		fn on_initialize(now: T::BlockNumber) -> Weight {
+			let mut dispatcher = T::QueuedMessageDispatch::with_any_weight();
+			let (queued_messages_dispatched, weight_spent_on_queued_dispatch) = dispatcher.dispatch();
+			if queued_messages_dispatched != 0 {
+				frame_support::debug::trace!(
+					target: "runtime",
+					"Spent {} weight on {} queued messages from on_initialize()",
+					weight_spent_on_queued_dispatch,
+					queued_messages_dispatched,
+				);
+			}
+
+			weight_spent_on_queued_dispatch
+		}
+
 		/// Send message over lane.
 		#[weight = 0] // TODO: update me (https://github.com/paritytech/parity-bridges-common/issues/78)
```

Comment:
`DELIVERY_BASE_WEIGHT` maybe?

Reply:
Not sure what do you mean. `DELIVERY_BASE_WEIGHT` is weight of completely another transaction on different chain

Reply:
It looked like they could be related. The comment has to do with a delivery transaction, which to me is something that `send_message()` would send.

Reply:
I still don't understand you - why `send_message()` sends delivery transaction?

Reply:
I didn't fully understand the `send` mechanics. Tomek cleared things up for me :)